app/src/main/java/app/grapheneos/apps/ui/MainActivity.kt

```diff
@@ -35,6 +36,16 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         val views = MainActivityBinding.inflate(layoutInflater)
         this.views = views
+        //get the current intent
+        val intent = intent
+
+        //get the attached extras from the intent
+        //we should use the same key as we used to attach the data.
+        val isSuW = intent.getBooleanExtra("SuW", false)
+
+        if (isSuW) {
```

Comment:
can just be a line.
`views.done.isVisible = isSuW`

Reply:
done